build/index.js

```diff
@@ -266,12 +267,40 @@ function logSuccess(command, timeLapsed) {
     logMain(message);
 }
 
+function checkVersions(metaManifest) {
+    const thisNPM = process.platform === "win32" ? "npm.cmd" : "npm";
```

Comment:
Nit: I guess I'd make this a global, calling it `npmCmd` or something.

Reply:
Yep, the problem is that it's actually a variable in the other case. I'll make it a function, maybe.